src/sentry/static/sentry/app/views/settings/projectAlerts/settings.jsx

```diff
@@ -77,30 +77,18 @@ class ProjectAlertSettings extends AsyncView {
       params: {orgId, projectId},
     } = this.props;
 
-    const basePath = `/settings/${orgId}/projects/${projectId}/alerts/`;
-
     return (
       <React.Fragment>
         <SettingsPageHeader
           title={t('Alerts Settings')}
           action={
-            <Button
-              to={`${basePath}new/`}
-              disabled={!canEditRule}
-              title={
-                !canEditRule
-                  ? t('You do not have permission to edit alert rules.')
-                  : undefined
-              }
-              priority="primary"
-              size="small"
-              icon={<IconAdd size="xs" isCircled />}
-            >
-              {t('New Alert Rule')}
+            <Button to={`/organizations/${orgId}/alerts/rules/`} size="small">
+              {t('View Alert Rules')}
```

Comment:
I was wondering about whether it'd be desired to retain the project filter here. Did you talk to Adhi or anyone else about this?

Reply:
we generally keep the project filter when moving between say the issues tab and discover, I think we should expect it to do the same here.